content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -184,18 +186,44 @@ otel:
     - b3
   resource:
     attributes:
-      environment: dev
-      xyz: foo
+      deployment.environment: dev
+      service:
+        name: cart
+        namespace: shop
 ```
 
 Environment variables example:
 
 ```shell
 export OTEL_PROPAGATORS="tracecontext,b3"
-export OTEL_RESOURCE_ATTRIBUTES="environment=dev,xyz=foo"
```

Comment:
It may also be great to have an example with a map value overrriden by an environment variable.

Reply:
that's exactly what we do here: change `deployment.environment` to `prod`.